Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml.cs

```diff
@@ -106,7 +108,7 @@ async void RunMBaaSAsync(object sender, EventArgs e)
                 var list = await Data.ListAsync<CustomDocument>(DefaultPartitions.UserDocuments);
                 foreach (var doc in list)
                 {
-                    AppCenterLog.Info(App.LogTag, "List result=" + JsonConvert.SerializeObject(doc));
+                    AppCenterLog.Info(App.LogTag, "List result=" + doc.Id);
```

Comment:
.JsonValue?

Reply:
This is just a change I made to find doc Ids, I'll revert it so it prints everything again.